c/include/libsbp/acquisition.h

```diff
@@ -37,130 +36,285 @@ SBP_PACK_START
  * acquisition search space with the best carrier-to-noise (CN/0)
  * ratio.
  */
-#define SBP_MSG_ACQ_RESULT         0x002F
-
 typedef struct SBP_ATTR_PACKED {
-  float cn0;    /**< CN/0 of best point [dB Hz] */
-  float cp;     /**< Code phase of best point [chips] */
-  float cf;     /**< Carrier frequency of best point [hz] */
-  sbp_gnss_signal_t sid;    /**< GNSS signal for which acquisition was attempted */
-} msg_acq_result_t;
+#define SBP_MSG_ACQ_RESULT 0x002F
+
+  /**
+   * CN/0 of best point [dB Hz]
+   */
+  float cn0;
+
+  /**
+   * Code phase of best point [chips]
+   */
+  float cp;
+
+  /**
+   * Carrier frequency of best point [hz]
+   */
+  float cf;
 
+  /**
+   * GNSS signal for which acquisition was attempted
+   */
+  sbp_gnss_signal_t sid;
+
+} msg_acq_result_t;
 
 /** Deprecated
  *
-* Deprecated.
+ * Deprecated.
  */
-#define SBP_MSG_ACQ_RESULT_DEP_C   0x001F
-
 typedef struct SBP_ATTR_PACKED {
-  float cn0;    /**< CN/0 of best point [dB Hz] */
-  float cp;     /**< Code phase of best point [chips] */
-  float cf;     /**< Carrier frequency of best point [hz] */
-  gnss_signal_dep_t sid;    /**< GNSS signal for which acquisition was attempted */
-} msg_acq_result_dep_c_t;
+#define SBP_MSG_ACQ_RESULT_DEP_C 0x001F
+
+  /**
+   * CN/0 of best point [dB Hz]
+   */
```

Comment:
this is objectively more pretty before than it is after.  It used to be visible in 5 lines, now it takes 25,.